x-pack/solutions/security/plugins/security_solution/public/timelines/components/timeline/body/helpers.tsx

```diff
@@ -111,12 +111,11 @@ export const isEvenEqlSequence = (event: Ecs): boolean => {
 };
 /** Return eventType raw or signal or eql */
 export const getEventType = (event: Ecs): Omit<TimelineEventsType, 'all'> => {
-  if (!isEmpty(event?.kibana?.alert?.rule?.uuid)) {
-    return 'signal';
-  } else if (!isEmpty(event?.eql?.parentId)) {
+  if (!isEmpty(event?.eql?.parentId)) {
     return 'eql';
-  }
-  return 'raw';
+  } else if (!isEmpty(event?.kibana?.alert?.rule?.uuid)) {
+    return 'signal';
+  } else return 'raw';
```

Comment:
the extra `else` is not needed but if you want to keep it, I'd recommend adding braces around it
```suggestion
  }
  return 'raw';
```

Reply:
This is done. Thanks.